src/audio/pipeline.c

```diff
@@ -111,7 +145,6 @@ static void connect_downstream(struct pipeline *p, struct comp_dev *start,
 
 	/* complete component init */
 	current->pipeline = p;
-	current->frames = p->ipc_pipe.frames_per_sched;
```

Comment:
ok, will look into it. I didn't think the testbench at all for this.

Reply:
ok, look into testbench, I don't see that fs_in or fs_out is actually used anywhere? with this patch If it is 0 it is set to 0 (again, because pipeline frames is always 0), if it is set from command line it is not touched. @ranj063 any thoughts?